.travis.yml

```diff
@@ -28,7 +28,7 @@ addons:
 env:
   global:
     - RUST_COMPILER_VERSION=stable
-    - EJB_RUST_BUILD_DIR="$TRAVIS_BUILD_DIR/exonum-java-binding-core/rust/"
+    - EJB_RUST_BUILD_DIR="$TRAVIS_BUILD_DIR/exonum-java-binding-parent/exonum-java-binding-core/rust/"
```

Comment:
It used to be used here, but then the code was moved into `run_travis_job.sh`. Shall this variable be defined there?

Reply:
yes. As for me `run_travis_job.sh` will be the better place for this variable